definitions.js

```diff
@@ -168,7 +168,7 @@ export const HOTEL_CLASS_FIVE_STARS = 'HOTEL_CLASS_FIVE_STARS';
 
 /* Purchases */
 // Food and beverages
-export const PURCHASE_CATEGORY_FOOD = 'Food';
+export const PURCHASE_CATEGORY_FOOD = 'FOOD';
```

Comment:
Is it okay to change this value? Thoughts @corradio?

A bit unsure due to this comment:
![image](https://user-images.githubusercontent.com/3296643/102367969-a3d45280-3fba-11eb-8544-9695171dffe3.png)

Reply:
If need be I can also change the name in the footprints.yml file back to `Food`

Reply:
Changing it requires double checking that the data has been migrated accordingly.
I don't think we have a migration system set up in Bloom right?

Reply:
Not for something like this, so I suggest reverting the change in footprints.yml as well :)

Reply:
Just FYI, here's the North code that reads the `oldKeys` values in footprint.yml and migrates the identifiers
https://github.com/tmrowco/north/blob/master/northapp/src/models.js#L522-L557

Reply:
I suggest we avoid changing the identifier here. That can be a separate task. Let us try to get this work wrapped up.